core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { NgClass, NgIf } from '@angular/common';
+import { Component, DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ButtonModule } from 'primeng/button';
+import { CardModule } from 'primeng/card';
+import { ScrollPanelModule } from 'primeng/scrollpanel';
+
+import {
+    TemplateBuilderBoxComponent,
+    TemplateBuilderBoxSize
+} from './template-builder-box.component';
+
+@Component({
+    selector: 'dotcms-host-component',
+    template: ` <dotcms-template-builder-box [size]="size"> </dotcms-template-builder-box>`
+})
+class TemplateBuilderBoxHostComponent {
+    size = 'large';
+}
+
+describe('TemplateBuilderBoxComponent', () => {
+    let component: TemplateBuilderBoxHostComponent;
+    let hostFixture: ComponentFixture<TemplateBuilderBoxHostComponent>;
+    let de: DebugElement;
+
+    beforeEach(async () => {
+        await TestBed.configureTestingModule({
+            imports: [
+                TemplateBuilderBoxComponent,
+                NgClass,
+                NgIf,
+                ButtonModule,
+                CardModule,
+                ScrollPanelModule
+            ],
+            declarations: [TemplateBuilderBoxHostComponent]
+        }).compileComponents();
+
+        hostFixture = TestBed.createComponent(TemplateBuilderBoxHostComponent);
+        component = hostFixture.componentInstance;
+        de = hostFixture.debugElement;
+        hostFixture.detectChanges();
+    });
+
+    it('should create the component', () => {
+        expect(component).toBeTruthy();
+    });
+
+    it('should render with default size', () => {
+        expect(de.nativeElement.querySelector('.template-builder-box__large')).toBeTruthy();
```

Comment:
We didn't tell you but we don't use styles selector for tests we do `data-testId="xxxx"` and then use that as a selector.

Also, Angular have a class to select elements: https://angular.io/api/platform-browser/By